Realm/Tests/Swift/SwiftSetTests.swift

```diff
@@ -150,59 +150,84 @@ class SwiftRLMSetTests: RLMTestCase {
         // Test int average
-        XCTAssertEqual(aggSet.set.average(ofProperty: "intCol")!.doubleValue, Double(10.0), accuracy: 0.1, "Average should be 1.0")
+        XCTAssertEqual(noArray.average(ofProperty: "intCol")!.doubleValue, Double(10.0), accuracy: 0.1, "Average should be 10.0")
+        XCTAssertEqual(yesArray.average(ofProperty: "intCol")!.doubleValue, Double(10.0), accuracy: 0.1, "Average should be 10.0")
 
         // Test float average
-        XCTAssertEqual(aggSet.set.average(ofProperty: "floatCol")!.doubleValue, Double(0.72), accuracy: 0.1, "Average should be 0.0")
+        XCTAssertEqual(noArray.average(ofProperty: "floatCol")!.doubleValue, Double(0.0), accuracy: 0.1, "Average should be 0.0")
+        XCTAssertEqual(yesArray.average(ofProperty: "floatCol")!.doubleValue, Double(1.2), accuracy: 0.1, "Average should be 1.2")
 
         // Test double average
-        XCTAssertEqual(aggSet.set.average(ofProperty: "doubleCol")!.doubleValue, Double(1.0), accuracy: 0.1, "Average should be 2.5")
+        XCTAssertEqual(noArray.average(ofProperty: "doubleCol")!.doubleValue, Double(2.5), accuracy: 0.1, "Average should be 2.5")
+        XCTAssertEqual(yesArray.average(ofProperty: "doubleCol")!.doubleValue, Double(0.0), accuracy: 0.1, "Average should be 2.5")
 
         // MIN ::::::::::::::::::::::::::::::::::::::::::::::
         // Test int min
-        var min = aggSet.set.min(ofProperty: "intCol") as! NSNumber
+        var min = noArray.min(ofProperty: "intCol") as! NSNumber
+        XCTAssertEqual(min.int32Value, Int32(10), "Minimum should be 10")
+        min = yesArray.min(ofProperty: "intCol") as! NSNumber
         XCTAssertEqual(min.int32Value, Int32(10), "Minimum should be 10")
 
         // Test float min
-        min = aggSet.set.min(ofProperty: "floatCol") as! NSNumber
+        min = noArray.min(ofProperty: "floatCol") as! NSNumber
         XCTAssertEqual(min.floatValue, Float(0), accuracy: 0.1, "Minimum should be 0.0f")
+        min = yesArray.min(ofProperty: "floatCol") as! NSNumber
+        XCTAssertEqual(min.floatValue, Float(1.2), accuracy: 0.1, "Minimum should be 1.2f")
 
         // Test double min
-        min = aggSet.set.min(ofProperty: "doubleCol") as! NSNumber
-        XCTAssertEqual(min.doubleValue, Double(0.0), accuracy: 0.1, "Minimum should be 1.5")
+        min = noArray.min(ofProperty: "doubleCol") as! NSNumber
+        XCTAssertEqual(min.doubleValue, Double(2.5), accuracy: 0.1, "Minimum should be 2.5")
+        min = yesArray.min(ofProperty: "doubleCol") as! NSNumber
+        XCTAssertEqual(min.doubleValue, Double(0.0), accuracy: 0.1, "Minimum should be 0.0")
 
         // Test date min
-        let dateMinOutput = aggSet.set.min(ofProperty: "dateCol") as! Date
-        XCTAssertEqual(dateMinOutput, dateMinInput, "Minimum should be dateMaxInput")
+        var dateMinOutput = noArray.min(ofProperty: "dateCol") as! Date
+        XCTAssertEqual(dateMinOutput, dateMaxInput, "Minimum should be dateMaxInput")
```

Comment:
For the assert message this could be  "Minimum should be \(dateMaxInput)", for all the date asserts

Reply:
Same as line 363 in 'SwiftArrayTests' file. Then is the code in the SwiftArrayTests file also wrong?

Reply:
I guess there is no need for this, it was just an extra

Reply:
Then, should I change the code of 'SwiftSetTests' file to the previous one?

Reply:
Nope don't worry.